requirements.txt

```diff
@@ -22,20 +22,21 @@ django-webtest==1.7.7
 factory_boy==2.5.2
 django-sslify>=0.2.0
 django-ckeditor==4.4.8
+wagtail==1.0b2
+django-sekizai==0.8.1
 
 # django-cms
-django-cms==3.0.7
-django-mptt==0.6
-django-sekizai==0.7
-django-classy-tags==0.5
-djangocms-admin-style==0.2.2
-html5lib==0.999
-six==1.3.0
-django-filer==0.9.5
-cmsplugin-filer==0.9.5
-django-reversion==1.7
-djangocms-text-ckeditor==2.4.1
-djangocms-picture==0.1
-djangocms-googlemap==0.2
-djangocms-file==0.1
-djangocms-video==0.1
+# django-cms==3.0.7
+# django-mptt==0.6
+# django-classy-tags==0.5
+# djangocms-admin-style==0.2.2
+# html5lib==0.999
+# six==1.3.0
+# django-filer==0.9.5
+# cmsplugin-filer==0.9.5
+# django-reversion==1.7
+# djangocms-text-ckeditor==2.4.1
+# djangocms-picture==0.1
+# djangocms-googlemap==0.2
+# djangocms-file==0.1
+# djangocms-video==0.1
```

Comment:
Might as well delete. You can always look on github to see previous version stuff.